### ISSUE TITLE
[Pattern] Revisit previous tests if the game has changed

### ISSUE THREAD

**Issue by garykac** [blinks/pattern#20]
- [ ] Confidence, from 1 to 3 stars.
- [ ] Context (followed by `* * *`)
- [ ] Problem (in bold)
- [ ] Forces
- [ ] Solution (in bold, followed by `* * *`)
- [ ] "How and why" explanatory text.

Just because an idea didn't work in the past (with an old version of your game) doesn't mean that same idea won't work with the current version of your game.

**Comment by blinks** [comment]
Yeah, I can see this one. Leads also to remembering the reasons you made certain changes / included certain rules. Also to slowly adding new playtesters.
